### DISCUSSION TITLE
How to deal with "Non-Strict YAML warning" for `.lima/_config/_networks.yaml`?YA

### DISCUSSION THREAD

**Discussion by ilyagr** [lima-vm/lima#2374] [Q&A] [answered]
I'm getting these warnings, which I believe come from https://github.com/lima-vm/lima/pull/1045:

> WARN[0000] Non-strict YAML is deprecated and will be unsupported in a future version of Lima: /Users/ilyagr/.lima/_config/networks.yaml  error="[17:3] unknown field \"vdeSwitch\"\n      14 | # socketVMNet has precedence over vdeVMNet.\n      15 |   socketVMNet: \"/opt/socket_vmnet/bin/socket_vmnet\"\n      16 | # vdeSwitch and vdeVMNet are DEPRECATED.\n    > 17 |   vdeSwitch: /opt/vde/bin/vde_switch\n             ^\n      18 |   vdeVMNet: /opt/vde/bin/vde_vmnet\n      19 |   varRun: /private/var/run/lima\n      20 |   sudoers: /private/etc/sudoers.d/lima\n      21 | "

I don't believe I created the `networks.yaml` file. What is the proper way to re-generate or otherwise fix it?

**Comment by jandubois** [comment, marked as answer]
> I'm getting these warnings, which I believe come from #1045:

The warnings are due to #2311 which removed the VDE related fields from `networks.yaml`.

> I don't believe I created the `networks.yaml` file. What is the proper way to re-generate or otherwise fix it?

You can just delete the file, and it will be recreated with the new default values.

**Reply by ilyagr** [reply]
Thanks, that worked fine!

Perhaps the warning for this specific file should suggest that the users can safely delete the file?

In any case, I'm happy.
